src/zip.js

```diff
@@ -28,5 +28,11 @@ module.exports = function(gj, options) {
         }
     });
 
-    return zip.generate({compression:'STORE'});
+    var generateOptions = { compression:'STORE' };
+
+    if (typeof window === 'undefined') {
```

Comment:
Can you use `process.browser` instead here? That's the more browserify'ish way to do it.